### PR TITLE
Testsuite addition

### PR DESCRIPTION
**Pull request by gino247**
Starting of introducing test-suite to project.
Test-suite: sinon.
If this is a **good** choice, then the next step will be to introduce **chai**, and **chai-as-Promise**, followed by building out the tests accordingly.

### CODE REVIEW

**Comment by alexcasalboni** [comment]
Sorry for the trouble @gino247 but could you please help me resolve the conflict from master so I can proceed with the review? 

I merged #86 before merging this, my apologies 🙏

File: test/unit/test-lambda.js [hunk 1]
```diff
@@ -863,28 +893,38 @@ describe('Lambda Functions', async() => {
             publishLambdaVersionCounter = 0;
```
Comment (alexcasalboni) [inline test/unit/test-lambda.js:893]: Same here (counters reset). Also the variables above are already defined, right?

File: test/unit/test-lambda.js [hunk 2]
```diff
@@ -50,72 +56,84 @@ const invokeForFailure = async(handler, event) => {
...
+    afterEach('Global mock utilities afterEach', () => {
+        // restore everything to it's natural order
+        sandBox.restore();
+    });
+
+    describe('initializer', async() => {
+
+        const handler = require('../../lambda/initializer').handler;
+
+        beforeEach('mock utilities', () => {
```
[the remark this answers is not in the document]
Comment (gino247) [reply test/unit/test-lambda.js:131]: Sorry for the wait, I will have a look this evening and push changes as needed.

File: test/unit/test-lambda.js (hunk 1 above)
Comment (gino247) [reply test/unit/test-lambda.js:893]: Sorry for the wait, I will have a look this evening and push changes as needed.

File: test/unit/test-lambda.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (gino247) [reply test/unit/test-lambda.js]: Done.

File: test/unit/test-lambda.js (hunk 1 above)
Comment (gino247) [reply test/unit/test-lambda.js]: Done.

**Comment by alexcasalboni** [comment]
Thanks @gino247 🙏

I've solved the conflict and pushed directly in your branch.

Merging this PR now 🎉

**Pull request merged**